addon/components/new-bourbon-select-field.js

```diff
@@ -38,6 +38,16 @@ export default Component.extend({
     );
   },
 
+  didInsertElement() {
```

Comment:
needed to add this to support https://github.com/MatchbookLabs/flabongo/blob/master/frontend/builder/app/components/inline-select.coffee#L19

and 

https://github.com/MatchbookLabs/flabongo/blob/master/frontend/builder/app/templates/components/builder-rules-rule-condition.emblem#L13